LXStudio-IDE/src/main/java/titanicsend/pattern/tmc/SimpleSolidEdgePattern.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * @author Travis Cline <travis.cline@gmail.com>
+ */
+
+package titanicsend.pattern.tmc;
+
+import heronarts.lx.LX;
+import heronarts.lx.LXCategory;
+import heronarts.lx.color.ColorParameter;
+import heronarts.lx.color.LXColor;
+import titanicsend.pattern.TEPattern;
+
+/**
+ * SimpleSolidPattern shows a trivial pattern that accepts input to control the output color pattern.
```

Comment:
It's got Edge in the name, but it colorizes all pixels, whether they're in an Edge or Panel, right? If that's on purpose, maybe rename? Or we could have a pattern that just colors the edges and use this name.

Reply:
right, fixing.

Reply:
changed to only set edgePoint colors